src/PolicyInsights/PolicyInsights/help/New-AzPolicyAttestation.md

```diff
@@ -0,0 +1,376 @@
+---
+external help file: Microsoft.Azure.PowerShell.Cmdlets.PolicyInsights.dll-Help.xml
+Module Name: Az.PolicyInsights
+online version:
+schema: 2.0.0
+---
+
+# New-AzPolicyAttestation
+
+## SYNOPSIS
+Creates a new policy attestation for a policy assignment.
+
+## SYNTAX
+
+### ByName (Default)
+```
+New-AzPolicyAttestation -Name <String> [-Scope <String>] [-ResourceGroupName <String>]
+ -PolicyAssignmentId <String> [-ComplianceState <String>] [-PolicyDefinitionReferenceId <String>]
+ [-ExpiresOn <DateTime>] [-Owner <String>] [-Comment <String>] [-Evidence <PSAttestationEvidence[]>]
+ [-AssessmentDate <DateTime>] [-Metadata <Object>] [-DefaultProfile <IAzureContextContainer>] [-WhatIf]
```

Comment:
What's the format for Metadata value? Object is an obscure object type

Reply:
@BethanyZhou The format for Metadata is JObject (JSON Object). I have wrapped it as a PSAttestationEvidence object. I will regerate this file to show the updated value. Thank you

Reply:
I will wrap it with PSAttestationMetadata type

Reply:
Is it applicable to use dictionary or something else to make this interface more friend to customer?

Reply:
@BethanyZhou Internally (on the server side), metadata is represented as a JSON object. So as long as one passes a valid JSON string, it should work fine. What would you recommend using instead such that we can make it more user friendly? Should we expose Metadata as a dictionary type and then parse it as a json object while processing it?

Reply:
Yes, that's what i'm thinking. what's your thought about this idea?

Reply:
I was thinking that I can make it a string, similar to how New-AzPolicyDefinition does it, here https://github.com/Azure/azure-powershell/blob/4668bff1cc4ed6510d9b0d4c6ec289ab27f04241/src/Resources/ResourceManager/Implementation/Policy/NewAzurePolicyDefinition.cs#L66. This parameter supports a string containing json content or a path to a json file.

Reply:
Sounds good

Reply:
@BethanyZhou Made the changes. Let me know if this works. I will further add a test or two to verify passing in a json file for metadata

Reply:
Looks good to me, please go ahead.